helper-cli/src/main/kotlin/commands/GeneratePackageConfigurationsCommand.kt

```diff
@@ -90,6 +91,10 @@ internal class GeneratePackageConfigurationsCommand : CliktCommand(
         val packageConfiguration = createPackageConfiguration(packageId, this)
         val outputFile = getOutputFile(filename)
 
+        if (outputFile.exists()) {
+            throw UsageError("The output file $outputFile must not exist yet.", statusCode = 2)
```

Comment:
Nit: We usually single-quote paths.

Reply:
I'd prefer to keep the behavior because I'm actually using it.
I'd have two alternatives which would probably satisfy both preferences
- if the file exists, load it and replace only the matcher bit (header)
- add a `--force` option to enable overwriting.

What do you think?

Reply:
I added the `--force-overwrite` option.